lib/trino-parquet/src/test/java/io/trino/parquet/TestTupleDomainParquetPredicate.java

```diff
@@ -136,9 +134,11 @@ public void testBoolean()
 
     private static BooleanStatistics booleanColumnStats(boolean minimum, boolean maximum)
     {
-        BooleanStatistics statistics = new BooleanStatistics();
-        statistics.setMinMax(minimum, maximum);
-        return statistics;
+        return (BooleanStatistics) Statistics.getBuilderForReading(Types.optional(PrimitiveTypeName.BOOLEAN).named(""))
```

Comment:
Please add some name like "BooleanColumn" instead of empty string here and in other places